ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -81,15 +80,13 @@ import { getCurrentChainId } from '../../../../shared/modules/selectors/networks
 ///: BEGIN:ONLY_INCLUDE_IF(solana-swaps)
 import { MultichainNetworks } from '../../../../shared/constants/multichain/networks';
 ///: END:ONLY_INCLUDE_IF
-import { trace, TraceName } from '../../../../shared/lib/trace';
```

Comment:
trace, TraceName were used with the receive button and modal which are no longer needed as we have removed the receive flow